recipes/lodepng/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "lodepng" %}
+{% set version = "20220109" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
```

Comment:
```suggestion

# The repo does not have tags, but the libraries uses a date-based versioning scheme that is tracked in comments and in the source code
```
Even though you are the only maintainer, other people may look at the recipe and wonder why it grabs archives via git hash.

Reply:
I added the change, but this is not sufficient on its own to make sure that the version is available to downstream CMake users. We also need to add a call to [write_basic_package_version](https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html#command:write_basic_package_version_file) and install the generated file.